test/actions/katello/organization_test.rb

```diff
@@ -42,22 +42,55 @@ def except_notification(action, action_result)
 
   class CreateTest < TestBase
     let(:action_class) { ::Actions::Katello::Organization::Create }
+    let(:organization) { FactoryBot.build(:organization, name: 'New Organization') }
+    let(:expected_cvv) { organization.default_content_view_version }
 
-    it 'plans' do
-      organization.expects(:create_library)
-      organization.expects(:create_anonymous_provider)
-      organization.expects(:create_redhat_provider)
-      organization.expects(:save!)
-      action.stubs(:action_subject).with(organization, any_parameters)
+    it 'plans and runs' do
+      organization.expects(:candlepin_owner_exists?).returns(false)
+      ::Organization.any_instance.expects(:load_debug_cert)
+      action.stubs(:action_subject).with(organization)
       plan_action(action, organization)
-      assert_action_planed_with(action,
+      assert_action_planned_with(action,
```

Comment:
:pray: 

We should remove that alias at some point..  :laughing:

Reply:
One day ..